sources/packages/backend/apps/api/src/main.ts

```diff
@@ -27,6 +28,12 @@ async function bootstrap() {
   const systemUsersService = app.get(SystemUsersService);
   await systemUsersService.loadSystemUser();
 
+  // Load dynamic form configurations.
+  const dynamicFormConfigurationService = app.get(
+    DynamicFormConfigurationService,
+  );
+  await dynamicFormConfigurationService.loadAllDynamicFormConfigurations();
+
```

Comment:
We followed this approach some times but I believe we can keep the `main` clean and achieve the same inside the module initialization implementing `OnModuleInit` in a similar way as it was done on [bull-board-queues-registration.module.ts](https://github.com/bcgov/SIMS/blob/7b8df04b35f01159deba96957544a42eea0eded6/sources/packages/backend/apps/queue-consumers/src/bull-board/bull-board-queues-registration.module.ts)